locust/env.py

```diff
@@ -33,6 +33,7 @@ def __init__(
         stop_timeout: float | None = None,
         catch_exceptions=True,
         parsed_options: Namespace | None = None,
+        parsed_locustfiles: list[str] = [],
```

Comment:
here's another dangerous default value :)

Reply:
Yeah I thought mypy was getting upset with it not being a list but just the type was incorrect